library/Octane/Type/CompressedWord.hs

```diff
@@ -43,7 +43,8 @@ $(OverloadedRecords.overloadedRecord Default.def ''CompressedWord)
 instance BinaryBit.BinaryBit CompressedWord where
   getBits n = do
     let limit = fromIntegral n
-    value <- getStep limit (bitSize limit) 0 0
+    let maxBits = bitSize limit
+    value <- getStep limit maxBits 0 0
```

Comment:
This is unnecessary. I bound it to a local value so I could print it out, but that's not necessary anymore.